demos/continuous_batching/README.md

```diff
@@ -53,7 +59,8 @@ node: {
   }
   node_options: {
       [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
-          models_path: "./"
+          models_path: "./",
+          cache_size: 50
```

Comment:
Should it be this big?

Reply:
This value was used in benchmarking script and the results are included. Default value might be to low. Which value do you suggest?

Reply:
16 maybe? For someone with 32 GBs RAM to be able to comfortably load it.

Reply:
I added a not to reduce it if there is not enough RAM. I would expect on hosts with Xeon at least 128GB on board. 
Do you think this is good enough?